README.md

```diff
@@ -76,103 +88,156 @@ Of course, this Guide can’t replace professional advice. But talk to anyone wh
-- ☝️🔹 However, there are subtleties to be aware of regarding what this outstanding total refers to:
-  - Private companies always have what is referred to as **“authorized but unissued”** shares. For example, a corporation might have 100 million authorized shares, but will only have actually issued 10 million shares. In this example, the corporation would have 90 million authorized but unissued shares. When you are trying to determine what percentage a number of shares represents, you do *not* make reference to the authorized but unissued shares.
-  - You actually want to know the total issued, but even this number can be confusing, as it can be computed [more than one way](http://www.mattbartus.com/option-grants-fully-diluted-or-issued-and-outstanding/). Typically, people refer to the total number of shares “issued and outstanding” or “fully diluted.”
-  - **“Issued and outstanding”** refers to the number of shares actually issued by the company to shareholders. Note this will not include shares that others may have an option to purchase.
-  - **“Fully diluted”** refers to all of the shares that have been issued, all of the shares that have been set aside in a stock incentive plan, and all of the shares that could be issued if all convertible securities (such as outstanding warrants) were exercised. A key difference is that this total will include all the shares in the employee option pool that are reserved but not yet issued to employees. (The option pool is discussed more below.)
-  - Generally, it’s best to know the fully diluted number to know the likely percentage a number of shares is worth in the future. The terminology mentioned here isn’t universally applied, either, so it’s worth discussing it to be sure there is no miscommunication.
+- [**Public companies**](https://en.wikipedia.org/wiki/Public_company) are corporations where any member of the public can own stock. People can buy and sell the stock for cash on public [exchanges](https://www.investopedia.com/terms/e/exchange.asp). The value of a company’s shares is the value you see in the stock market reports, so shareholders know how much their stock is worth.
+- Most smaller companies, including all startups, are [**private companies**](https://en.wikipedia.org/wiki/Privately_held_company), where shareholders are restricted, meaning you can’t buy and sell stock easily or to anyone at any time.
+- ❗️ It’s hard to know precisely the value of private company stock. A stock certificate is a piece of paper that entitles you to something of highly uncertain value, and could well be worthless in the future, or highly valuable, depending on the fate of the company. Because the value of private company stock is not determined by public trades, shareholders make educated guesses about the likely future value, at a time when they will be able to buy or sell the stock.
```

Comment:
This wording is confusing to me. I think I know what we're trying to get at here, though. 
* While each share in a company has a par value at any given point in time, a holder of that stock may not be able to exchange that share for cash. 
* Despite having value "on paper (maybe a good time to define what it means to have value on paper)," stock can end up being worthless.

Reply:
I think the salient point here is that it's not really possible in private company because there is no stock market value circling buildings in red or green lights in Times Square. So even if you *could* exchange for cash, you would have a tough time determining how much cash you'd get in exchange for the paper representation of changing value.

Reply:
Revised in upcoming edits:

- ❗️ It’s hard to know the “real value” of private company stock.
  Because the value of private company stock is not determined by public trades,
  shareholders can only make educated guesses about the likely future value, at a time when
  they will be able to buy or sell the stock.
  It’s always worth remembering stock is a legal agreement that entitles you to something of
  highly uncertain value, and could well be worthless in the future, or highly valuable,
  depending on the fate of the company.
  - 👆 We’ll discuss the notion of [fair market value](#409a-valuations) later, but even if a
    company assigns a value to the stock for tax and accounting purposes, it doesn’t mean you
    can expect to sell it for that value!